interface_specifications/pe_openapi/apis/v1/d1_pe_definition_statuses_put.md

```diff
@@ -0,0 +1,38 @@
+### Status Notification to Verifier
+
+`PUT <verifier-example.io>pe/definitions/{definition_id}/statuses`
+
+
+#### Description
+
+This is an optional interaction and hence party can choose not to implement it. The PE REST Backend invokes an HTTP REST API exposed by the party to notify about the status of exchange.
+
+#### Parameters
+
+Path Variables:
+* `definition_id`: definition id e.g. `32f54163-7166-48f1-93d8-ff217bdb0653`
+
+Request Body
+
+```json
+{
+  "thread":{
+    "id": "f1ca8245-ab2d-4d9c-8d7d-94bf310314ef"
+  },
+  "definition_id": "32f54163-7166-48f1-93d8-ff217bdb0653",
```

Comment:
Is there a specific reason why we are repeating the definition_id which is in the path already in the body?

Reply:
We need this definition id to be present in the object to validate it.
We can indeed copy the id present in the thread to the object on the backend side, in controller or filters.

So, shall we remove the redundancy ?

Reply:
Yes. It is always a bit awkward to have redundancy in your input data

Reply:
ok removing from request body.